spec/mail/encoding_spec.rb

```diff
@@ -115,13 +116,14 @@
     it "should allow you to send in unencoded strings to fields and encode them" do
       mail = Mail.new
       mail.charset = 'ISO-8859-1'
-      subject = "This is あ string"
+      subject = String.new("This is あ string")
       subject.force_encoding('ISO8859-1') if RUBY_VERSION > '1.9'
       mail.subject = subject
       result = mail[:subject].encoded
-      string = "Subject: =?ISO-8859-1?Q?This_is_=E3=81=82_string?=\r\n"
+      string = String.new("Subject: =?ISO-8859-1?Q?This_is_=E3=81=82_string?=\r\n")
       if RUBY_VERSION > '1.9'
         string.force_encoding('ISO8859-1')
+        result = result.dup if result.frozen?
         result.force_encoding('ISO8859-1')
```

Comment:
Having to use `String.new` everywhere is feeling off.

How about using literal string syntax, instead, and duping as needed:
```ruby
  string = string.dup.force_encoding('ISO8859-1')
  result = result.dup.force_encoding('ISO8859-1')
```

Then we can use standard syntax up-front and skip the `#frozen?` checks.